example/Example.elm

```diff
@@ -33,16 +36,26 @@ porterConfig =
 type alias Model =
     { porter : Porter.Model String String Msg
     , response : String
+    , advanced_response : String
     }
 
 
 init : ( Model, Cmd Msg )
 init =
     ( { porter = Porter.init
       , response = ""
+      , advanced_response = ""
       }
-      -- Send a request through porter, specifying the response handler directly
-    , Porter.send Receive "Reverse me!" |> Cmd.map PorterMsg
+    , Cmd.batch
+        [ -- Send a request through porter, specifying the response handler directly
+          Porter.send porterConfig Receive "Reverse me!"
+
+        -- Or send multiple requests one after the other:
+        , Porter.request ("Reverse me too!")
+            |> Porter.andThen (\reversed_str -> Porter.request (reversed_str ++ " The Quick Brown Fox!"))
+            |> Porter.andThen (\reversed_str -> Porter.request (reversed_str ++ " A man a plan a canal: panama"))
+            |> Porter.sendRequest porterConfig Receive2
```

Comment:
I think it would make it easier for folks to make the links if this were named `ReceiveAdvanced`